tinyquery/runtime.py

```diff
@@ -377,7 +372,8 @@ def check_types(self, arg):
 
     def _evaluate(self, num_rows, column):
         # TODO: Use CityHash.
-        values = map(pass_through_none(hash), column.values)
+        hash_fn = pass_through_none(hash)
+        values = [hash_fn(x) for x in column.values]
```

Comment:
I think `column.values` is basically a row?  So the var might be better called `r` than `x`, though it's not a big deal either way.  (Both here and various places below.)